src/SEOManager.php

```diff
@@ -309,7 +312,11 @@ public function __get(string $key): string|null
         return $this->get(Str::snake($key, '.'));
     }
 
-    /** Handle magic set. */
+    /**
+     * Handle magic set.
+     *
+     * @phpstan-return string|array|null
```

Comment:
Can we just use `@return` here? I'm generally trying to avoid any static analysis-specific annotations in code whenever possible. Ideal case scenario is only configuring things in the phpstan config file, and having the code agnostic of the static analysis.

Reply:
Done.